dotenv/load.ts

```diff
@@ -1,5 +1,12 @@
 // Copyright 2018-2022 the Deno authors. All rights reserved. MIT license.
 
-import { configSync } from "./mod.ts";
+import { loadSync } from "./mod.ts";
 
-configSync({ export: true });
+if (!(Deno.readTextFileSync instanceof Function)) {
+  // deno deploy doesn't have Deno.readFileSync
+  console.warn(
+    `Deno.readTextFileSync is not a function: No .env data was read.`,
+  );
+} else {
+  loadSync();
+}
```

Comment:
I don't understand this bit, so automatic loading is always synchronous?

Reply:
> automatic loading is always synchronous?

Yes, otherwise synchronous top-level `Deno.env.get()` call happens before loading variables from `.env`

We fixed that in #1964